tests/backup/backup_basic_test.go

```diff
@@ -150,6 +150,12 @@ var _ = BeforeSuite(func() {
 	} else {
 		log.Infof("Locked bucket name not provided")
 	}
+	log.Infof("Dumping source and destination kubeconfig to file system path")
+	kubeconfigs := os.Getenv("KUBECONFIGS")
+	dash.VerifyFatal(kubeconfigs != "", true, "Getting KUBECONFIGS Environment variable")
+	kubeconfigList := strings.Split(kubeconfigs, ",")
+	dash.VerifyFatal(len(kubeconfigList), 2, "2 kubeconfigs are required for source and destination cluster")
+	DumpKubeconfigs(kubeconfigList)
```

Comment:
My bad. I had suggested to place it in BeforeSuite. But I think it makes sense if it is in `BackupInitInstance`

Reply:
Corrected .
Running with latest change :https://jenkins.pwx.dev.purestorage.com/job/Users/job/Sumit/job/Custom-Pipelines/job/px-backup-on-demand-system-test-byoc/833/console

Reply:
Latest log : https://jenkins.pwx.dev.purestorage.com/job/Users/job/Sumit/job/Custom-Pipelines/job/px-backup-on-demand-system-test-byoc/834/consoleFull